patches/tModLoader/Terraria/ModLoader/IO/TagSerializer.cs

```diff
@@ -51,7 +51,7 @@ public abstract class TagSerializer
 			return false;
 		}
 
-		public static void AddSerializer(TagSerializer serializer) {
+		internal static void AddSerializer(TagSerializer serializer) {
 			serializers.Add(serializer.Type, serializer);
```

Comment:
So um, should we search tML itself for `ILoadable` ? I'm seeing more and more use for such a thing.

Reply:
I most definitely wanted autoloading in the default mod for at least the content, yes. Make an issue.